solution/ui/regulations/js/api.js

```diff
@@ -243,6 +243,29 @@ const getLastParserSuccessDate = async (apiURL, { title = "42" }) => {
     return result.end ? niceDate(result.end.split("T")[0]) : "N/A";
 };
 
+/**
+ * Get array of objects containing valid GovInfo docs years with links to the PDF files.
+ *
+ * @param {string} apiURL - URL of API passed in from Django.  Ex: `/v2/` or `/v3/`
+ * @param {Object} params - parameters needed for API call
+ * @param {string} params.title - CFR title number.
+ * @param {string} params.part - CFR part numer within title.
+ * @param {string} params.[("section"|"appendix"|"subpart")] - CFR idenfifier for node type.  Ex. for "section": "10"
+ *
+ * @returns {Array<{year: string, link: string}>}
+ */
+const getGovInfoLinks = async (apiURL, params) => {
+    // manually adjust to v3 if needed
+    const url = apiURL.replace("/v2/", "/v3/");
```

Comment:
Just a note not a you need to do this.  This whole changing the api url to v2 to v3 should not have to be done anymore.  I understand we are converting but most of our apis are v3 not v2.  If anyhing it should be the other way around